src/main/features/dashboard/views/index.njk

```diff
@@ -1,6 +1,6 @@
 {% extends "layout.njk" %}
 {% from "form.njk" import submitButton, errorSummary %}
-{% from "responseStatus.njk" import responseStatusForDefendant, responseStatusForClaimant %}
+{% from "./macros/status.njk" import statusForDefendant, statusForClaimant %}
```

Comment:
3 different ways of importing this macro :(

Reply:
i was having issues with it and got one that worked and just passed my mind. Updating now

Reply:
changed